README.md

```diff
@@ -31,57 +31,99 @@ Using a theme, all of your default configuration lives in an npm package.
-  + [Product Card Grid](#product-card-grid)
-  + [Resource Card](#resource-card)
-  + [Carousel](#carousel)
-  + [Embedding markdown documents and filtering content](#embedding-markdown-documents-and-filtering-content)
-* [Customizations](#customizations)
-  + [Custom Layout](#custom-layout)
-  + [Frame](#frame)
-  + [Theming](#theming)
-* [Upgrading](#upgrading)
-* [Issue tracker](#issue-tracker)
-* [Contributing](#contributing)
-* [Releases](#releases)
+- [Adobe I/O Theme](#adobe-io-theme)
+  - [Contents](#contents)
+  - [Getting started](#getting-started)
+    - [Using GitHub repository templates](#using-github-repository-templates)
+    - [Using the Adobe I/O CLI](#using-the-adobe-io-cli)
+  - [Content structure](#content-structure)
+    - [Markdown pages](#markdown-pages)
+    - [Links](#links)
+      - [Internal links](#internal-links)
+      - [External links](#external-links)
+    - [Assets](#assets)
+    - [Google Docs support](#google-docs-support)
+      - [Relative links](#relative-links)
+      - [Metadata](#metadata)
+      - [Exclude folders or documents](#exclude-folders-or-documents)
+  - [Configuration](#configuration)
+    - [Environment variables](#environment-variables)
+      - [GitHub Contributors](#github-contributors)
+      - [Analytics and Feedback component](#analytics-and-feedback-component)
+      - [Identity Management Services](#identity-management-services)
+  - [Algolia indexing and search](#algolia-indexing-and-search)
+    - [Server-side environment variables](#server-side-environment-variables)
+    - [Client-side (browser) environment variables](#client-side-browser-environment-variables)
+  - [Global Navigation](#global-navigation)
+  - [Menus](#menus)
+  - [Home link](#home-link)
+  - [Side Navigation](#side-navigation)
+    - [Variations](#variations)
+      - [Single-level side navigation](#single-level-side-navigation)
+      - [Single-level side navigation with headers](#single-level-side-navigation-with-headers)
+      - [Multi-level side navigation](#multi-level-side-navigation)
+      - [Auto-collapsing of multi-level side navigation](#auto-collapsing-of-multi-level-side-navigation)
+      - [Use descriptive titles](#use-descriptive-titles)
+      - [Be concise](#be-concise)
+      - [Use sentence case](#use-sentence-case)
+      - [Use the right variation](#use-the-right-variation)
+      - [Avoid deep nested menus](#avoid-deep-nested-menus)
+      - [Use consistent multi-level behavior](#use-consistent-multi-level-behavior)
+  - [Versions](#versions)
+  - [Building the site](#building-the-site)
+    - [Adding a Path Prefix](#adding-a-path-prefix)
+  - [Deploying the site](#deploying-the-site)
+    - [Preview on GitHub Pages](#preview-on-github-pages)
+    - [Deploy to Azure Storage Static Websites](#deploy-to-azure-storage-static-websites)
+  - [Writing Enhanced Markdown](#writing-enhanced-markdown)
+    - [Metadata with Front matter](#metadata-with-front-matter)
+  - [<pre>](#pre)
```

Comment:
The summary is messed up. I think you can revert the summary changes since you're not adding a new section ?

Reply:
fixed